emperor/support_files/js/color-view-controller.js

```diff
@@ -132,16 +132,26 @@ define([
           var data = decompViewDict.setCategory(attributes, scope.setPlottableAttributes, category);
 
           if (scaled) {
-            scope.setSlickGridDataset({});
-            scope.$gridDiv.hide();
+            //build plottables as all non-numerics
+            var nonNumeric = uniqueVals.filter(function(v) { return isNaN(v)});
+            var plotList = data.filter(function(x) {
+              return $.inArray(x.category, nonNumeric) !== -1;
+            });
+            var plottables = [];
+            for (var i = 0; i < plotList.length; i++) {
+              plottables = plottables.concat(plotList[i].plottables);
+            }
+            // Set SlickGrid for color of non-numeric values and show color bar for rest
+            scope.setSlickGridDataset([{category: 'Non-numeric values', value:'#64655d', plottables: plottables}]);
+            scope.$gridDiv.css('height', '30px');
```

Comment:
I need to have a specific height for a single row of the slickgrid, so you can still change the color of the non-numeric values. I will dig into slickgrid to see if I can pull that height dynamically, unless you know the command.